public/favicons/safari-pinned-tab.svg

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
You should at least compress the svg 😄 
https://jakearchibald.github.io/svgomg/

Reply:
Added a new instruction to compact files on the generator and updated the files.

Reply:
Nice, so by activating the compress option we get to compress the svg file as well as minimal compression on rasterized images.